pytest_fixtures/component/satellite_auth.py

```diff
@@ -45,7 +45,7 @@ def ldap_cleanup():
 def ad_data():
     supported_server_versions = ['2016', '2019']
```

Comment:
we can remove 2016 from the list here.

Reply:
I'm not sure why. It's still somewhat supported in robottelo, we're just missing an instance and are not using it currently.

Reply:
okay got it.